cockroachcloud/quickstart.md

```diff
@@ -219,80 +226,42 @@ Use the [JDBC driver](https://jdbc.postgresql.org/) in a Java application.
-1. Use the `cockroach convert-url` command to convert the connection string that you copied earlier to a [valid connection string for JDBC connections](../{{site.versions["stable"]}}/connect-to-the-database.html?filters=java):
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    cockroach convert-url --url "<connection-string>"
-    ~~~
+In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied in Step 2 above:
 
-    ~~~
-    ...
-
-    # Connection URL for JDBC (Java and JVM-based languages):
-    jdbc:postgresql://{host}:{port}/{database}?options=--cluster%3D{routing-id}&password={password}&sslmode=verify-full&user={username}
-    ~~~
-
-1. Set the `JDBC_DATABASE_URL` environment variable to the JDBC-compatible connection string:
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    export JDBC_DATABASE_URL="<jdbc-connection-string>"
-    ~~~
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export JDBC_DATABASE_URL="<jdbc-connection-string>"
+~~~
 
-    The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
+The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
 </section>
 
 <section class="filter-content" markdown="1" data-scope="windows">
-1. Use the `cockroach convert-url` command to convert the connection string that you copied earlier to a [valid connection string for JDBC connections](../{{site.versions["stable"]}}/connect-to-the-database.html?filters=java):
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    cockroach convert-url --url "<connection-string>"
-    ~~~
-
-    ~~~
-    ...
-
-    # Connection URL for JDBC (Java and JVM-based languages):
-    jdbc:postgresql://{host}:{port}/{database}?options=--cluster%3D{routing-id}&password={password}&sslmode=verify-full&user={username}
-    ~~~
-
-1. Set the `JDBC_DATABASE_URL` environment variable to the JDBC-compatible connection string:
+In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied in Step 2 above:
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    $env:JDBC_DATABASE_URL = "<jdbc-connection-string>"
-    ~~~
+{% include_cached copy-clipboard.html %}
+~~~ shell
+$env:JDBC_DATABASE_URL = "<jdbc-connection-string>"
+~~~
 
-    The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
+The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
 </section>
 
+{{site.data.alerts.callout_success}}
+For reference information about connecting to CockroachDB with supported client drivers, see [Connect to a CockroachDB Cluster](../../docs/stable/connect-to-the-database.html).
```

Comment:
```suggestion
For information about connecting to CockroachDB with supported client drivers, see [Connect to a CockroachDB Cluster](../../docs/stable/connect-to-the-database.html).
```

Reply:
This text is copied from database-url-environment-variable.md
![image](https://user-images.githubusercontent.com/24593519/176732485-427dcf5d-99b8-42f3-9591-3c393702cb47.png)

and used for the node, python, and go languages already. This simply adds the same message to the java code. 
![image](https://user-images.githubusercontent.com/24593519/176732752-6eb1c876-8227-418c-88bf-770662871ef1.png)

Changing this here would require me to also change the original code.

Reply:
no prob, fine to leave as-is!